src/test/java/com/github/kokorin/jaffree/ffmpeg/FFmpegTest.java

```diff
@@ -211,6 +212,43 @@ public void testDuration() throws Exception {
         outputDuration = getDuration(outputPath);
         assertEquals(10.0, outputDuration, 0.1);
     }
+    
+    @Test
+    public void testDurationInstance() throws Exception {
+        Path tempDir = Files.createTempDirectory("jaffree");
+        Path outputPath = tempDir.resolve(Artifacts.VIDEO_MP4.getFileName());
+
+        FFmpegResult result = FFmpeg.atPath(Config.FFMPEG_BIN)
+                .addInput(UrlInput
+                        .fromPath(Artifacts.VIDEO_MP4)
+                        .setDuration(Duration.ofSeconds(10))
+                )
+                .addOutput(UrlOutput
+                        .toPath(outputPath)
+                        .copyAllCodecs())
+                .execute();
+
+        Assert.assertNotNull(result);
+
+        double outputDuration = getDuration(outputPath);
+        assertEquals(10.0, outputDuration, 0.1);
+
+        result = FFmpeg.atPath(Config.FFMPEG_BIN)
+                .addInput(UrlInput
+                        .fromPath(Artifacts.VIDEO_MP4)
+                        .setDuration(Duration.ofMinutes(1).dividedBy(6))
+                )
+                .setOverwriteOutput(true)
+                .addOutput(UrlOutput
+                        .toPath(outputPath)
+                        .copyAllCodecs())
+                .execute();
+
+        Assert.assertNotNull(result);
+
+        outputDuration = getDuration(outputPath);
+        assertEquals(10.0, outputDuration, 0.1);
```

Comment:
This test is useless, i should test Jaffress's functionality and only it. But it repeats code above with the only difference in `Duration.ofMinutes(1).dividedBy(6)`.
Pay attention, that other methods are not tested.

Reply:
@kokorin this test assures that the functionality is the same functionality as the test it was copied from.
Meaning it assures that either of these tests fails we can see if the other one did it. If it didn't then we can be assured it works.

On top of that the test assures that the Duration class gives the expected output back compared to the other implementation.

Reply:
@kokorin just to ask:
Since the test is useless, should i delete it? Or should i copy it to the other 2?
Because all this test does is assure that the output is consistent with other methods.

One test basically validates them for all 3.
Please tell me what you want since you gave conflicting instructions:
(Delete because useless but please add them to the other 2 anyways?)

Reply:
> One test basically validates them for all 3.

It does not: you added 3 methods and each method should be tested.

Reply:
> It does not: you added 3 methods and each method should be tested.

Yeah and the implementation of all 3 are exactly the same. But I will do it.

Reply:
The test method runs the same test twice. 
Effectively the second time only checks that `Duration.ofSeconds(10)` is the same as `Duration.ofMinutes(1).dividedBy(6)`.
It's tested in Java Time, so no need for the 2nd run. 
Please, delete it.